docs/features/gamification.md

```diff
@@ -1,4 +1,4 @@
-![HPI Logo](img/HPI_Logo.png)
+![HPI Logo](img/../HPI_Logo.png)
```

Comment:
Documentation file 'features/gamification.md' contains a link to 'features/HPI_Logo.png' which is not found in the documentation files.

Reply:
fixed